bot/utils/checks.py

```diff
@@ -135,7 +135,7 @@ async def predicate(cog: Cog, ctx: Context) -> None:
         if any(role.id in bypass for role in ctx.author.roles):
             return
 
-        # cooldown logic, taken from discord.py internals
+        # cooldown logic, taken from disnake's, internals
```

Comment:
Reviewing every line paid off
```suggestion
        # cooldown logic, taken from disnake's internals
```

Reply:
hah, nice spot https://github.com/python-discord/bot/pull/2103/commits/445c4a533eb11e9f9c8d8b438c858f7425775d50